frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1955,10 +2022,83 @@ function ReaderHighlight:extendSelection()
         pos0 = new_pos0,
         pos1 = new_pos1,
         pboxes = new_pboxes,
+        ext = ext,
     }
     UIManager:setDirty(self.dialog, "ui")
 end
 
+-- Calculates positions, text, pboxes of one page of selected multi-page highlight
+-- (reflow mode must be off)
+function ReaderHighlight:getExtendedHighlightPage(pos0, pos1, cur_page)
+    local item = {}
```

Comment:
May be, to make it clear when one reads that off this PR: `if not self.ui.paging then return end`. (Same for the next function below?)